### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 9th of May 2023 17:30 CET / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#1014] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 9th of May 2023 17:30 CEST / 11:30am EDT / 8:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [here](https://www.youtube.com/watch?v=zl1UGb3oI4E)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- @t-lo update regarding `scripts` merge of `coreos-overlay` and `portage-stable`.
- @jepio  KubeCon Amsterdam update - [watch his talk here](https://www.youtube.com/watch?v=rb08ubfJQSM&ab_channel=CNCF%5BCloudNativeComputingFoundation%5D) 
- @t-lo Brief discussion of our Release Planning revamp 

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks
- Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- Close this discussion as "Resolved".

**Comment by t-lo** [comment]
I'll provide an overview of the recent `scripts` merge of `coreos-overlay` and `portage-stable`.

**Comment by t-lo** [comment]
@jepio can we get you interested in doing a quick kubecon recap?

**Reply by jepio** [reply]
sure thing!
